### ISSUE TITLE
Compare beta diversity between and within groups

### ISSUE THREAD

**Issue by microsud** [microsud/microbiomeutilities#149]
Request for function that allows comparison of beta diversity (boxplots) between and within groups.

**Comment by antagomir** [comment]
The `microbiome::divergence` function can already be used for this purpose, at least to some extent. We can develop it further, though. Some examples in the [tutorial](https://microbiome.github.io/tutorials/Betadiversity.html).

**Comment by microsud** [comment]
Yes I think the `microbiome::divergence` function can be wrapped for more intuitive and global comparison (`compare_divergence`/`global_divergence`) to get something as shown below. I got some request for this. IMO the appropriate place for this is `microbiome` pkg.

![compare_beta](https://user-images.githubusercontent.com/16476233/97102704-08082380-16a8-11eb-860e-2987fd6ca139.png)

**Comment by antagomir** [comment]
Good to me. PRs are welcome, or we may implement it when time allows.
